qiskit/visualization/pulse/matplotlib.py

```diff
@@ -572,35 +692,48 @@ def _draw_channels(self, ax, output_channels, interp_method, t0, tf, dt, scale_d
+             channel_scales: Dict['Channel', float] = None,
+             channels_to_plot: List['Channel'] = None,
+             plot_all: bool = True, table: bool = True,
+             label: bool = False, framechange: bool = True,
+             scaling: float = None, channels: List['Channel'] = None,
+             show_framechange_channels: bool = True
+             ) -> 'matplotlib.figure.Figure':
         """Draw figure.
 
         Args:
-            schedule (ScheduleComponent): Schedule to draw
-            dt (float): time interval
-            interp_method (Callable): interpolation function
-                See `qiskit.visualization.interpolation` for more information
-            plot_range (tuple[float]): plot range
-            scale (float): Relative visual scaling of waveform amplitudes.
-            channel_scales (dict[Channel, float]): Channel independent scaling as a
-                dictionary of `Channel` object.
-            channels_to_plot (list[OutputChannel]): deprecated, see `channels`
-            plot_all (bool): if plot all channels even it is empty
-            table (bool): Draw event table
-            label (bool): Label individual instructions
-            framechange (bool): Add framechange indicators
-            scaling (float): Deprecated, see `scale`
-            channels (list[OutputChannel]): channels to draw
-            show_framechange_channels (bool): Plot channels with only framechanges
+            schedule: schedule object to plot.
+            dt: Time interval of samples. Pulses are visualized in the unit of
+                cycle time if not provided.
+            interp_method: Interpolation function. See example.
+                Interpolation is disabled in default.
+                See `qiskit.visualization.pulse.interpolation` for more information.
+            plot_range: A tuple of time range to plot.
+            scale: Scaling of waveform amplitude. Pulses are automatically
+                scaled channel by channel if not provided.
+            channel_scales: Dictionary of scale factor for specific channels.
+                Scale of channels not specified here is overwritten by `scale`.
+            channels_to_plot: Deprecated, see `channels`.
+            plot_all: When set `True` plot empty channels.
+            table: When set `True` draw event table for supported commands.
+            label: When set `True` draw label for individual instructions.
+            framechange: When set `True` draw framechange indicators.
+            scaling: Deprecated, see `scale`.
+            channels: A list of channel names to plot.
+                All non-empty channels are shown if not provided.
+            show_framechange_channels: When set `True` plot channels
+                with only framechange instructions.
 
         Returns:
-            matplotlib.figure: A matplotlib figure object for the pulse schedule
+            matplotlib.figure.Figure: A matplotlib figure object
```

Comment:
```suggestion
            A matplotlib figure object
```
this could be one line now
```suggestion
            A matplotlib figure object for the pulse envelope.
```
but the next line also has to be removed